src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -512,7 +529,22 @@ class ChatSwitcherView extends React.Component {
                     onRemoveFromGroup={this.removeUserFromGroup}
                     onConfirmUsers={this.startGroupChat}
                 />
-
+                <View style={[
+                    styles.flexRow,
+                    styles.sidebarHeaderTop,
+                    styles.justifyContentBetween,
+                    styles.alignItemsCenter,
+                ]}
+                >
+                    <Text numberOfLines={2} style={[styles.headerText, styles.textLarge]}>
+                        Chats
+                    </Text>
```

Comment:
This is perhaps a good case to re-use the `Header` component

https://github.com/Expensify/Expensify.cash/blob/ca2fa88a5789b82463d35eddc3d57f70a7286868/src/components/Header.js#L12-L18

Reply:
It would also be nice if we could pass a `textSize` prop to `Header` with options of `default`, or `large` for the text size change if you are OK with this. 

cc @shawnborton

Reply:
Actually, Great idea. I overlooked the header component. In the future, we can add more things if needed so it is a good idea to put it there.